src/game/rendering_graph_node.c

```diff
@@ -644,7 +652,7 @@ Lights1 defaultLight = gdSPDefLights1(
     0x3F, 0x3F, 0x3F, 0xFF, 0xFF, 0xFF, 0x00, 0x00, 0x00
 );
 
-Vec3f globalLightDirection = { 0x28, 0x28, 0x28 };
+Vec3f globalLightDirection = { 0x49, 0x49, 0x49 };
```

Comment:
This shouldn't be changed unconditionally. Although, what is our HLE solution for this to make sure the game doesn't look wacky on most emulator plugins?

Reply:
Those values are equivalent on standard microcode, since it's just the same thing but normalized. F3D/EX/EX2 just normalizes them behind the scenes, while EX3 does not.

Reply:
Does this mean F3DEX2 still works with these values?

Reply:
Yes.